File: 3153/3153.cs
public class Solution {
    public int SumOfDigitDifferencesOfAllPair(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(SumOfDigitDifferencesOfAllPair(root.left), SumOfDigitDifferencesOfAllPair(root.right));
    }
}

File: 3153/3153.cpp
class Solution {
public:
    int sumOfDigitDifferencesOfAllPair(TreeNode* root) {
        if(!root) return 0;
        return 1+max(sumOfDigitDifferencesOfAllPair(root->left),sumOfDigitDifferencesOfAllPair(root->right));
    }
};